jvm/src/test/scala/tastyquery/BaseUnpicklingSuite.scala

```diff
@@ -54,16 +54,19 @@ abstract class BaseUnpicklingSuite(withClasses: Boolean, withStdLib: Boolean, al
     (base, classRoot)
   }
 
-  def resolve(path: DeclarationPath)(implicit ctx: BaseContext): Symbol =
-    followPath(defn.RootPackage, path)
+  /* TODO Currently this only resolves symbols that have already been loaded,
+   * but the intent is that it will load symbols when required.
+   */
+  def resolve(path: DeclarationPath)(using BaseContext): Symbol =
```

Comment:
this resolve won't work to find nested packages, because packages are entered with their fully qualified name

Reply:
ok, you changed the definition of lookup